### PR TITLE
depr: Drop Python 3.10

### PR DESCRIPTION
**Pull request by speller26**

### CODE REVIEW

**Review by virajvchaudhari — DISMISSED** [review]

**Review by rmshaffer — DISMISSED** [review]

File: tests/providers/test_adapter.py [hunk 1]
```diff
@@ -1654,10 +1654,3 @@ def test_missing_qubit_in_properties_handled_gracefully(self):
 
         # Verify the target still works with remaining qubits
         self.assertEqual(target.num_qubits, len(MOCK_RIGETTI_TOPOLOGY_GRAPH.nodes))
-
-    def test_empty_circuit_with_barrier(self):
-        """Test edge case: empty Qiskit circuit with global barrier."""
-        circuit = QuantumCircuit(0)
-        circuit.barrier()
-        braket_circuit = to_braket(circuit)
-        self.assertEqual(len(braket_circuit.instructions), 0)
```
Comment (virajvchaudhari) [inline tests/providers/test_adapter.py:1663]: why we are removing this as part of this PR?

Comment (speller26) [reply tests/providers/test_adapter.py:1663]: This test case is invalid as of https://github.com/amazon-braket/amazon-braket-sdk-python/pull/1203

**Review by virajvchaudhari — APPROVED** [review]
